teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization, name = 'Unknown', age = nil)
+    super(name, age)
```

Comment:
- [ ] This constructor is missing the `parent_permission` of the `Person` class. kindly add it up 👍

Reply:
I added **parent_permission** in the constructor of the Person class:

![fix-02](https://github.com/jlcarrascof/OOP-school-library/assets/86327867/41d3cab3-dc1b-4d78-91fc-3a02b87fbd40)